### PR TITLE
Update  diem version.

### PR DESCRIPTION
**Pull request by RIg410**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,24 +208,24 @@ See help:
...
 
 See help:
 
 ```
-disassembler --help
+decompiler --help
 ```
 
-Try to disassembly `.mv` file:
+Try to decompiler `.mv` file:
```
Comment (fzzr-) [inline README.md:225]: Typo. There's should be "Try to __decompile__ `.mv` file:"

**Review by fzzr- — APPROVED** [review]
👍

**Comment by villesundell** [comment]
Hello everyone, and thank you for updating to 1.2.0 so quickly!

However, just verifying, if this is the desired change: `dove run` needs `--signers` supplied for every script now (this used not to be the case). Is this desired? I suppose in default scenario(s) it's redundant information, since the address is available in Dove.toml?

(It's not possible to create issues in this repository, so that's why I am asking this here :slightly_smiling_face: )

**Comment by borispovod** [comment]
Please, update README (installation commands etc), before merge

**Comment by borispovod** [comment]
Hi @villesundell , thank you question!

We are busy with update, @fzzr- or @RIg410 will answer you shortly.

**Pull request merged**